sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/implementation/CommunicationBearerTokenCredential.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.jobrouter.implementation;
+
+import com.azure.communication.common.CommunicationTokenCredential;
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.credential.TokenRequestContext;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * This class serves as a CommunicationTokenCredential wrapper that
+ * allows using BearerAuthenticationPolicy in different clients
+ */
+public class CommunicationBearerTokenCredential implements TokenCredential {
```

Comment:
Do we need this class? I believe we're only using HMAC auth.

Reply:
No, will remove.